core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/catalog/CamelModelFactory.java

```diff
@@ -78,7 +79,7 @@ public static void initializeModels(HashMap<String, CamelModel> mockedSupportedC
 	 * @param majorMinorVersion	a version string in format major.minor (2.17)
 	 * @return	the full supported camel version (2.17.1.redhat-630xxx) or null if not supported
 	 */
-	public static String getCamelVersionFor(String majorMinorVersion) {
+	private static String getCamelVersionFor(String majorMinorVersion) {
```

Comment:
Yeah, I refactored the code to no longer use that method and instead use the getCompatibleCamelVersion. Seems I just forgot to remove it :) Good catch.